### PR TITLE
Remove `browsers/compare/ie` references [fixes #14142]

### LINKED ISSUE #14142: Redirect firefox/IE comparison page
This page is out of date and not a significant source of conversions. It can be removed and redirect to the Edge comparison.

- [x] Remove /firefox/browsers/compare/ie/ and associated assets
- [ ] Remove links to it from around the site (compare subnav, look for others)
- [ ] Redirect /firefox/browsers/compare/ie/ to /firefox/browsers/compare/edge/

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
We have outdated information on Internet Explorer on our comparison pages, so it's time to remove the page and references and assets to it.

## Significant changes and points to review
- Removed `compare/ie/index.html` and `compare/ie.ftl` files from codebase
- Removed assets and links leading to the page
- Removed references of IE from the `firefox/browsers/compare/index.html` page

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/14142

## Testing

- [ ] http://localhost:8000/en-US/firefox/browsers/compare/
- [ ] http://localhost:8000/en-US/firefox/browsers/compare/ie
- [ ] http://localhost:8000/en-US/firefox/more

### CODE REVIEW

**Comment by reemhamz** [comment]
Just waiting on an image asset to be updated before it's ready for review

File: l10n/en/firefox/browsers/compare/index.ftl [hunk 1]
```diff
@@ -27,15 +26,27 @@ compare-index-how-well-does-your-browser = How well does your browser work acros
...
-compare-index-find-out-how-firefox-fallback = Find out how { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera } and { -brand-name-ie } differ in terms of privacy, utility and portability.
+
+compare-index-find-out-how-firefox-chrome-v2 = Find out how { -brand-name-firefox }, { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera } and { -brand-name-brave } differ in terms of privacy, utility and portability.
+
+# Obsolete string
 compare-index-looking-for-a-better-browser = Looking for a better browser? We’ll compare { -brand-name-firefox } with { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera }, { -brand-name-brave } and { -brand-name-ie } to help you make your decision.
-compare-index-looking-for-a-better-fallback = Looking for a better browser? We’ll compare { -brand-name-firefox } with { -brand-name-chrome }, { -brand-name-edge }, { -brand-name-safari }, { -brand-name-opera } and { -brand-name-ie } to help you make your decision.
```
Comment (reemhamz) [inline l10n/en/firefox/browsers/compare/index.ftl:33]: I removed some older `-fallback` FTL string IDs that are now being replaced w/ new fallbacks. These were 4+ years old so should be fine to remove

**Review by craigcook — CHANGES_REQUESTED** [review]
nice work! Just one other card on /more and we can consider IE dead and buried.

File: bedrock/firefox/templates/firefox/more.html [hunk 2]
```diff
@@ -232,14 +232,6 @@ <h2>{{ ftl('learn-more-about-firefox', fallback='firefox-products-are') }}</h2>
         link_url=url('firefox.browsers.compare.edge')
       )}}
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/firefox/templates/firefox/more.html:234]: 🦅 👁️

**Review by craigcook — APPROVED** [review]

File: bedrock/firefox/templates/firefox/more.html [hunk 3]
```diff
@@ -232,14 +232,6 @@ <h2>{{ ftl('learn-more-about-firefox', fallback='firefox-products-are') }}</h2>
...
 
-      {{ card(
-        title=ftl('comparing-firefox-ie'),
-        ga_title='Compare IE',
-        image=resp_img('img/firefox/more/firefox-ie.jpg', optional_attributes={"loading": "lazy", "class": "mzp-c-card-image"}),
```
Comment (janbrasna) [inline bedrock/firefox/templates/firefox/more.html:238]: The `'media/img/firefox/more/firefox-ie.jpg'` is still around but it seems it was only ever linked from here. Should it be removed too?

@reemhamz I'm gonna PR pngcrushed versions of some of the changed images here, so I can include the removal of this stray asset with it if it's OK.

**Pull request merged**